WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -345,12 +348,28 @@ private async Task CreateRoundsAsync(CancellationToken cancellationToken)
 		{
 			var feeRate = (await Rpc.EstimateSmartFeeAsync((int)Config.ConfirmationTarget, EstimateSmartFeeMode.Conservative, simulateIfRegTest: true, cancellationToken).ConfigureAwait(false)).FeeRate;
 
-			RoundParameters roundParams = new(Config, Network, Random, feeRate, Config.CoordinationFeeRate);
+			RoundParameters roundParams = new(Config, Network, Random, feeRate, Config.CoordinationFeeRate, GetMaxSuggestedAmount(ConnectionConfirmationStartedCounter));
 			Round r = new(roundParams);
 			Rounds.Add(r);
 		}
 	}
 
+	internal static Money GetMaxSuggestedAmount(int roundCounter)
+	{
+		Money baseAmount = Money.Coins(0.1m);
+
+		return roundCounter switch
+		{
+			0 => baseAmount,
+			var n when (n % 32 == 0) => baseAmount * 100000,
```

Comment:
Using a constant doesn't look correct because there exists a limit already that is smaller than this so, it should be the minimum of this and the max allowed input.

Reply:
Done.